NAMESPACE

```diff
@@ -1,6 +1,7 @@
 # Generated by roxygen2: do not edit by hand
 
 export(model_importance)
+export(split_data_by_task)
```

Comment:
Does this function need to be exported? Generally you should only export functions that an end-user would call. Along the same lines, maybe `validate_input_data()` also does not need to be exported?